satpy/readers/seviri_base.py

```diff
@@ -353,6 +353,62 @@
                          'ALPHA': 0.9981,
                          'BETA': 0.5635}}
 
+# Calibration coefficients from Meirink, J.F., R.A. Roebeling and P. Stammes, 2013:
+# Inter-calibration of polar imager solar channels using SEVIRI, Atm. Meas. Tech., 6,
+# 2495-2508, doi:10.5194/amt-6-2495-2013
+
+# To obtain the slope for the calibration, one should use the routine get_seviri_meirink_slope
+
+# Epoch for the MEIRINK re-calibration
+DATE_2000 = datetime(2000, 1, 1)
+
+MEIRINK_COEFS = {}
+
+# Meteosat-8
+
+MEIRINK_COEFS[321] = {'VIS006': (24.346, 0.3739),
+                      'VIS008': (30.989, 0.3111),
+                      'IR_016': (22.869, 0.0065)
+                      }
+
+# Meteosat-9
+
+MEIRINK_COEFS[322] = {'VIS006': (21.026, 0.3739),
+                      'VIS008': (26.875, 0.3111),
+                      'IR_016': (21.394, 0.0065)
```

Comment:
Typo: Met-09 offsets should be `0.2556, 0.1835, 0.0498`

Reply:
oooops, thanks for the proofreading!